documents-webapp/src/main/webapp/vue-app/documents/components/body/actions/CopyLinkMenuAction.vue

```diff
@@ -30,7 +30,16 @@ export default {
     copyLink() {
       this.loading = true;
       let path;
-      if (this.isFileEditable())  {
+      if (this.file.folder){
+        path = `${window.location.host}${eXo.env.portal.context}`;
```

Comment:
It should may be user the whole `window.location.origin` including the protocol, WDYT ?

```suggestion
        path = `${window.location.origin}${eXo.env.portal.context}`;
```